wrapper/build.gradle.kts

```diff
@@ -124,15 +124,42 @@ tasks.spotbugsTest {
     }
 }
 
-tasks.jacocoTestCoverageVerification {
+tasks.withType<JacocoCoverageVerification> {
     violationRules {
         rule {
             limit {
-                // Coverage verification will pass if it is greater than or equal to 1%.
-                minimum = "0.01".toBigDecimal()
+                minimum = BigDecimal(0.57)
             }
         }
     }
+
+    afterEvaluate {
+        classDirectories.setFrom(files(classDirectories.files.map {
+            fileTree(it).apply {
+                exclude(
+                        "software/amazon/jdbc/wrapper/*",
+                        "software/amazon/jdbc/util/*",
+                        "software/amazon/jdbc/profile/*",
+                        "software/amazon/jdbc/plugin/DataCacheConnectionPlugin*"
```

Comment:
Why not to include all plugin package?

Reply:
This code excludes certain classes from the test coverage. We have tests for some of the more important plugins, like EFM and Failover.